Etabs_Adapter/CRUD/Create/Material.cs

```diff
@@ -143,7 +158,130 @@ private eMatType MaterialTypeToCSI(MaterialType materialType)
+
+            m_model.PropMaterial.GetOConcrete_1(
+                material.DescriptionOrName(),
+                ref fc,
+                ref lw,
+                ref fcsFactor,
+                ref sstype,
+                ref sshystype,
+                ref strainAtFc,
+                ref strainUltimate,
+                ref finalSlope,
+                ref frictionAngle,
+                ref dilationAngle
+                );
+
+            success &= (0 == m_model.PropMaterial.SetOConcrete_1(
+                material.DescriptionOrName(),
+                material.CylinderStrength,
+                IsLightweight(material),
+                IsLightweight(material) ? 0.75 : 1.0, //Strength reduction factor for lightweight concrete
+                sstype,
+                sshystype,
+                strainAtFc,
+                strainUltimate,
+                finalSlope,
+                frictionAngle,
+                dilationAngle
+                ));
+
+            return success;
+        }
+
+        /***************************************************/
 
+        private bool SetDesignMaterial(Steel material)
+        {
+            bool success = true;
+
+            double fy = 0;
+            double fu = 0;
+            double Efy = 0;
+            double Efu = 0;
+            int sstype = 0;
+            int sshystype = 0;
+            double strainAtHardening = 0;
+            double strainAtMaxStress = 0;
+            double strainAtRupture = 0;
+            double finalSlope = 0;
+
+            m_model.PropMaterial.GetOSteel_1(
+                material.DescriptionOrName(),
+                ref fy,
+                ref fu,
+                ref fy,
+                ref fu,
+                ref sstype,
+                ref sshystype,
+                ref strainAtHardening,
+                ref strainAtMaxStress,
+                ref strainAtRupture,
+                ref finalSlope
+                );
+
+            success &= (0 == m_model.PropMaterial.SetOSteel_1(
+                material.DescriptionOrName(),
+                material.YieldStress,
+                material.UltimateStress,
+                Efy,
+                Efu,
+                sstype,
+                sshystype,
+                strainAtHardening,
+                strainAtMaxStress,
+                strainAtRupture,
+                finalSlope
+                ));
+
+            return success;
+        }
+
+        /***************************************************/
+
+        private bool SetDesignMaterial(IMaterialFragment material)
+        {
+            Engine.Base.Compute.RecordError($"Could not set ETABS design parameters for material: {material.DescriptionOrName()}. Please set them manually in ETABS, or load the material from ETABS database prior to push.");
+            return true;
+        }
+
+        /***************************************************/
+
+        private bool IsLightweight(Concrete material)
```

Comment:
This is raising a warning for _all_ non-american concrete materials. Not sure it is suitable as this warning might just lead to confusion.

We might need to either define a LightWeight parameter on the Concrete material itself, or find some other solution to this.

Reply:
Agreed, need some more parameters on Structural materials - similar to what we've done on Timber.

Reply:
As below, will only raise the warning on concrete that appears lightweight (either name contains LW or density is less than 2080 kg/m^3 (~130pcf)